main/lsp/request_dispatch.cc

```diff
@@ -104,11 +89,6 @@ LSPResult LSPLoop::processRequestInternal(unique_ptr<core::GlobalState> gs, cons
                 logger->error(errorInfo->message);
             }
             return LSPResult{move(gs), {}};
-        } else if (method == LSPMethod::SorbetShowOperation) {
```

Comment:
I added this before so I could delegate sending show operation notifications from LSPPreprocessor. Now, LSPPreprocessor can send notifications directly to the client, so I don't need this code.